documentation/docs/decentralized_identifiers/update.mdx

```diff
@@ -12,19 +12,382 @@ keywords:
+The created document only contains one verification method with [capabilityInvocation](https://www.w3.org/TR/did-core/#capability-invocation) relationship.
+This method is used to sign the document, effectively controlling the DID.
+
+Any future updates to the document in this example will be signed using this verification method. The Account takes care of this automatically so we don't need to think about signing each update.
+
+### 2. Adding Verification Methods
+
+We can add another verification method to the DID document using the Account.
+
+<Tabs>
+<TabItem value="Rust" label="Rust">
+
+```rust
+account
+    .update_identity()
+    .create_method()
+    .content(methodcontent::generateed25519)
+    .fragment("my-next-key")
+    .apply()
+    .await?;
+```
+
+</TabItem>
+<TabItem value="node" label="Node.js">
+
+```js
+await account.createMethod({
+    content: MethodContent.GenerateEd25519(),
+    fragment: "my-next-key"
+})
+```
+</TabItem>
+</Tabs>
+
+The code above creates a new verification method that includes an Ed25519 public key, 
+signs the updated document using the private key of the default `capabilityInvocation` verification method 
+and publishes the document to the tangle.
+
+The updated Document will look as follows:
+
+
+```json
+{
+   "doc":{
+      "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf",
+      "verificationMethod":[
+         {
+            "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf#my-next-key",
+            "controller":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf",
+            "type":"Ed25519VerificationKey2018",
+            "publicKeyMultibase":"z2Zthec5siTfxCjPwZUHGDGybKNy9oc3ZYeftvEE2nEL3"
+         }
+      ],
+      "capabilityInvocation":[
+         {
+            "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf#sign-0",
+            "controller":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf",
+            "type":"Ed25519VerificationKey2018",
+            "publicKeyMultibase":"z5k7vzMVuXXj8MJDcfzP2owvc8xKBA6BBsAkFf1GSNu2X"
+         }
+      ]
+   },
+   "meta":{
+      "created":"2022-04-13T09:27:48Z",
+      "updated":"2022-04-13T09:28:06Z"
+   }
+}
+```
+
+:::note
+
+Setting the [fragment](https://www.w3.org/TR/did-core/#fragment) is required in order to uniquely identify the verification method so we can reference it in the future. 
+
+:::
+
+### 3. Adding Verification Relationships
+"A verification relationship expresses the relationship between the DID subject and a verification method.
+
+Different verification relationships enable the associated verification methods to be used for different purposes. 
+It is up to a verifier to ascertain the validity of a verification attempt by checking that the verification method used is contained in the appropriate verification 
+relationship property of the DID Document." [[w3c](https://www.w3.org/TR/did-core/#verification-relationships)]
+
+Verification Relationships include:
+- Authentication
```

Comment:
Can we add a very short 1-liner to explain each of them and add individual W3C links instead of the single link down below?